components/automate-ui/src/app/pages/data-feed-create/data-feed-create.component.html

```diff
@@ -0,0 +1,232 @@
+<div ngClass="{slide : isSlideOpen}">
+  <div class="sidenav-header-data-feed">
+    <div *ngIf="integrationSelected">
+      <chef-button class="arrow-btn" secondary (click)="returnToMenu()" data-cy="close-revision-id-button">
+        <chef-icon>arrow_back</chef-icon>
+      </chef-button>
+    </div>
+    
+    <h1 slot="title" class="bolder-font" *ngIf="!integrationSelected">Choose a Data Feed Integration</h1>
+    <h1 slot="title" class="bolder-font" data-cy="title-integration" *ngIf="integrationSelected">{{integTitle}}</h1>
+    <chef-button class="close" secondary (click)="closeCreateSlider()" data-cy="close-revision-id-button">
+      <chef-icon>close</chef-icon>
+    </chef-button>
+  </div>
+  <app-notification
+    *ngIf="testErrorGetter"
+    type="error"
+    timeout="5"
+    (dismissed)="dismissNotification()">
+    <span *ngIf="authSelected=='Username and Password'">Unable to connect: check URL, username and password.</span>
+    <span *ngIf="authSelected=='Access Token'">Unable to connect: check Token Type (Prefix) and token.</span>
+  </app-notification>
+  <app-notification
+    *ngIf="testSuccessGetter"
+    type="info"
+    timeout="5"
+    (dismissed)="dismissNotification()">
+    Notification test connected successfully!
+  </app-notification>
+  <main *ngIf="!integrationSelected" data-cy="interation-menu">
+    <div class="flex-container">
+        <h2 class="bolder-font">Webhook Integration</h2>
+        <div class="flex-child">
+          <div class="card" (click)="selectIntegration(item.name)" *ngFor="let item of integrations.webhook" [attr.data-cy]="item.name">
+            <img src="/assets/img/{{item.asset}}.png" alt="Avatar">
+            <div class="container image-footer">
+              <h3 class="bolder-font">{{item.name}}</h3> 
+            </div>
+          </div>
+        </div>
+    </div>
+    <div class="flex-container">
+        <h2 class="bolder-font">Storage Integration </h2>
+        <div class="flex-child">
+          <div class="card" (click)="selectIntegration(item.name)" *ngFor="let item of integrations.storage">
+            <img src="/assets/img/{{item.asset}}.png" alt="Avatar">
+            <div class="container image-footer">
+              <h3 class="bolder-font">{{item.name}}</h3> 
+            </div>
+          </div>
+        </div>
+    </div>
+  </main>
+  <main [class.hide]="!integrationSelected">
+    <div class="flex-container2">
+      <form [formGroup]="createForm" data-cy="data-feed-create-form">
+        <div class="input-margin">
+          <chef-form-field>
+            <label>
+              <span class="label">Name <span aria-hidden="true">*</span></span>
+              <input
+                chefInput
+                #name
+                formControlName="name"
+                data-cy="add-name"
+                type="text"
+                id="name-input"
+                autocomplete="off"/>
+            </label>
+            <chef-error
```

Comment:
indentation is not correct for chef-error

Reply:
done